src/components/PlanterDetail.js

```diff
@@ -195,11 +193,29 @@ const PlanterDetail = (props) => {
               </Typography>
             </Grid>
             <Divider />
+            <Grid container direction="column" className={classes.box}>
+              <Typography variant="subtitle1">Country</Typography>
+              <Typography variant="body1">
+                {(planterRegistration &&
+                  planterRegistration
+                    .map((item) => item.country)
+                    .filter(
+                      (country, i, arr) =>
+                        country && arr.indexOf(country) === i,
+                    )
+                    .join(', ')) ||
+                  '---'}
+              </Typography>
+            </Grid>
+            <Divider />
             <Grid container direction="column" className={classes.box}>
               <Typography variant="subtitle1">Registered</Typography>
               <Typography variant="body1">
                 {(planterRegistration &&
-                  getDateTimeStringLocale(planterRegistration.createdAt)) ||
+                  getDateTimeStringLocale(
+                    planterRegistration[planterRegistration.length - 1]
```

Comment:
We aren't guaranteed that the last registration in the array will be the earliest.
We could either order by registration date in the API query or perform a sort when the results are returned.

Reply:
You're right there. I will implement a sort on the frontend.